README.md

```diff
@@ -0,0 +1,30 @@
+# Ahoy::Intercom
+
+## Installation
+
+Add this line to your application's Gemfile:
+
+```ruby
+gem 'ahoy-intercom'
+```
+
+And then execute:
+
+    $ bundle
+
+Or install it yourself as:
+
+    $ gem install ahoy-intercom
+
+## Usage
+
+Set your environment variables `ENV['INTERCOM_APP_ID']` and `ENV['INTERCOM_ACCESS_TOKEN']`.
```

Comment:
I added link to doc
https://github.com/mkdev-me/ahoy-intercom/pull/1/files#diff-04c6e90faac2675aa89e2176d2eec7d8R22